src/Search/Api/Elasticsearch/Command/BuildIndexCommand.php

```diff
@@ -59,15 +63,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $create = null;
 
         // Try removing old one.
-        try {
-            $delete = $this->client->deleteIndex();
-        } catch (Throwable $e) {
-            $logger->debug($e->getMessage(), $e->getTrace());
+        if ($toDelete) {
+            try {
+                $delete = $this->client->deleteIndex();
+            } catch (Throwable $e) {
+                $logger->debug($e->getMessage(), $e->getTrace());
```

Comment:
The issue I've seen is on a clean environment where there is no index in the first instance, and an error is displayed saying "no index to delete"

If you're okay with the errors showing under those circumstances I can add it